app/lib/frontend/templates/listing.dart

```diff
@@ -83,6 +87,23 @@ String renderPackageList(List<PackageView> packages) {
   });
 }
 
+/// Renders the `views/pkg/liked_package_list.mustache` template.
+String renderMyLikedPackagesList(List<LikeData> likes) {
+  final packagesJson = [];
+  for (final like in likes) {
+    final package = like.package;
+    packagesJson.add({
+      'url': urls.pkgPageUrl(package),
+      'name': package,
+      'liked_date': shortDateFormat.format(like.created),
+      'thumb_up_outlined': staticUrls.assets['img__thumb-up-24px_svg'],
```

Comment:
You don't need these, we always have access to `{{{static_assets.img__thumb-up-filled-24px_svg}}}` in templates..